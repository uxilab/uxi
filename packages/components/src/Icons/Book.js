import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Book = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 18.57 15.74"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M0,13.05a.82.82,0,0,0,.82.82H7.07l1.64,1.64a.83.83,0,0,0,1.15,0l1.64-1.64h6.25a.82.82,0,0,0,.82-.82V.82A.82.82,0,0,0,17.76,0H11.17a.82.82,0,0,0-.58.24L9.29,1.54,8,.24A.82.82,0,0,0,7.4,0H.82A.82.82,0,0,0,0,.82ZM10.1,3l1.4-1.4h5.43v10.6H11.17a.82.82,0,0,0-.58.24L10.1,13ZM1.63,1.63H7.07L8.47,3V13L8,12.47a.82.82,0,0,0-.58-.24H1.63Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Book.propTypes = {
  color: PropTypes.string,
};

Book.defaultProps = {
  color: '#6d6d71',
};

export default Book;
