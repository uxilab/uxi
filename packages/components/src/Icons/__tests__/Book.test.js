import React from 'react';
import Book from '../Book';

describe('Icon : <Book />', () => {
  it('match snapshot', () => {
    expect(shallow(<Book />)).toMatchSnapshot();
  });
});

