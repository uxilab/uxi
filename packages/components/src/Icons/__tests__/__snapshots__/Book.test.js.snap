// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Book /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 18.57 15.74"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M0,13.05a.82.82,0,0,0,.82.82H7.07l1.64,1.64a.83.83,0,0,0,1.15,0l1.64-1.64h6.25a.82.82,0,0,0,.82-.82V.82A.82.82,0,0,0,17.76,0H11.17a.82.82,0,0,0-.58.24L9.29,1.54,8,.24A.82.82,0,0,0,7.4,0H.82A.82.82,0,0,0,0,.82ZM10.1,3l1.4-1.4h5.43v10.6H11.17a.82.82,0,0,0-.58.24L10.1,13ZM1.63,1.63H7.07L8.47,3V13L8,12.47a.82.82,0,0,0-.58-.24H1.63Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
